client/src/linkManager.js

```diff
@@ -198,6 +198,8 @@ export class linkManager extends LuigiClientBase {
    * @param {Object} updatedModalSettings possibility to update the active modal.
    * @param {Object} updatedModalSettings.title update the `title` of the active modal.
    * @param {Object} updatedModalSettings.size update the `size` of the active modal.
+   * @param {string} updatedModalSettings.width lets you specify a precise width for the modal. Allowed units are 'px', '%', 'rem', 'em', 'vh' and 'vw'.
```

Comment:
```suggestion
   * @param {string} updatedModalSettings.width update the `width` of the modal. Allowed units are 'px', '%', 'rem', 'em', 'vh' and 'vw'.
```
I'd make the same change for the others if this is OK.

Reply:
I updated all places as you suggested - please verify